Wrong modifier on background colors

[official docs](http://tachyons.io/docs/themes/skins/)
[source](https://github.com/tachyons-css/tachyons/blob/master/src/_skins.css)

`--color-name` should be `-color-name`

```css
.black { color: #000000 }
.bg-light-silver { background-color: #AAAAAA }
```

Trying to build the docs on macOS Sierra I get:
```bash
> ./docs2json.sh

[ERROR] couldn't read from 'docs/':
EISDIR: illegal operation on a directory, read
```
🤔

--- a/app/assets/data.json
+++ b/app/assets/data.json
@@ -38,7 +38,7 @@
     "bg"
   ],
   "modifiers": [
-    "--color-name"
+    "-color-name"
   ],
   "media": null,
   "tags": [

--- a/docs/background-colors.yml
+++ b/docs/background-colors.yml
@@ -10,7 +10,7 @@ base:
   - bg
 
 modifiers:
-  - --color-name
+  - -color-name
 
 media:
 

